LibreNMS/Alert/Transport/Messagebirdvoice.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/**
+ * Messagebirdvoice.php
+ *
+ * LibreNMS Messagebird voice API Tranport
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * @link       https://www.librenms.org
+ *
+ * @copyright  2023 Sjef van Zeeland
+ * @author     https://github.com/jepke/
+ *
+ * Messagebird voice will return 201 status if the call was initiated, if status is not 201 LibreNMS will log the full error
+ *
+ * @contributer f0o, sdef2
+ * Thanks to F0o <f0o@devilcode.org> for creating the Slack transport which is the majority of this code.
+ * Thanks to sdef2 for figuring out the differences needed to make Discord work.
+ * Thanks to theherodied for discord transport used as a base for messagebird.
+ */
+
+namespace LibreNMS\Alert\Transport;
+
+use LibreNMS\Alert\Transport;
+use LibreNMS\Exceptions\AlertTransportDeliveryException;
+use LibreNMS\Util\Http;
+
+class Messagebirdvoice extends Transport
+{
+    public function deliverAlert(array $alert_data): bool
```

Comment:
You might want to specify transport name on this to make it more readable in lists.  I think Teams does this, so look there.

Reply:
Didnt know but yeah much better like this